data.json

```
@@ -2208,6 +2208,15 @@
                 "JavaScript"
             ],
             "description": "a react-based framework that provides accessibility, modularity, responsiveness, and theming in a tidy package"
-        }
+        },
+        {
+            "name": "Todo List App",
+            "link": "https://github.com/pixelledbot/todo-list-app",
+            "label": "good-first-issue",
```

```suggestion
            "label": "good first issue",
```

Most projects use `good first issue` without the dashes. If that is the case with this entry, please commit this suggested change.